doc/README.md

```diff
@@ -198,14 +198,14 @@ paths with the highest cum weight.
 
 *  **_cum_ value**, the value of a location plus all of its descendants, is indicated by the nodes' color.
```

Comment:
Maybe describe the flat value first? It goes as the first column in the pprof text reports and I think in general in profiler documentation self / flat is described first, then total / cum is discussed.

Reply:
Done.